library/src/main/java/io/andref/rx/widgets/ListViewCard.java

```diff
@@ -183,8 +182,18 @@ private void layoutViews()
                     Log.e(TAG, "Drawable resource not found: " + exception.getMessage());
                 }
 
-                itemObservables.add(
-                        RxView.clicks(view)
+                RxView.clicks(view)
+                    .map(new Func1<Void, Item>()
+                    {
+                        @Override
+                        public Item call(Void aVoid)
+                        {
+                            return item;
+                        }
+                    })
+                    .subscribe(mItemClicks);
```

Comment:
Same as above. Plus we're creating a subscription for every view.  Seems unnecessary.